src/traces/scattergl/index.js

```diff
@@ -863,9 +863,11 @@ function plot(container, subplot, cdata) {
 
                 fillOptions.opacity = trace.opacity;
                 fillOptions.positions = pos;
+
+                return fillOptions;
```

Comment:
Can you briefly explain why this fixes https://github.com/plotly/plotly.js/issues/2376

Reply:
That fixes #2354. That is internal `regl-line2d` behavior: when we do `line2d.update([opts1, undefined, opts2, ...rest])`, the second line in batch will be ignored instead of removed. I made it accept `null` as argument, indicating that batch item should be removed.